scripts/rstats.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+#!coding:utf-8
+
+# Copyright 2022 TiKV Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import re
+import json
+import argparse
+
+def main():
+    pattern = r'.*SlowLog.*'
```

Comment:
This pattern may not precise enough, it should be `.*SlowLog:.*'

Reply:
addressed.